src/Recommendations/Process/Hydration.php

```diff
@@ -50,8 +56,11 @@ private function getModel(RuleModel $model, $unwrap = false)
             return $entity->wait(true);
         }
 
-        return $entity->then(function ($model) {
-            if (method_exists($model, 'getType')) {
+        return $entity->then(function (Model $model) {
```

Comment:
The goal was to have this perform a conversion from our `RuleModel<type, id>` and turn it into an API-SDK model, although we are only using it for Articles currently